build-tools/automation/azure-pipelines.yaml

```diff
@@ -180,6 +180,15 @@ stages:
         configuration: $(XA.Build.Configuration)
         packDirectory: $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)
 
+    - task: NuGetCommand@2
+      displayName: push Xamarin.Android.Sdk
+      continueOnError: true
+      inputs:
+        command: push
+        packagesToPush: '$(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)/*.nupkg'
+        nuGetFeedType: external
+        publishFeedCredentials: 'xamarin-impl public feed'
```

Comment:
@pjcollins @kzu are we really pushing this to Nuget? Or is this to an internal pipelines feed?

Reply:
We are pushing this via nuget to an undisclosed feed used for CI/temp/internal stuff that's not really critical or important to keep in a private secure feed (much like the VSSDK stuff is all available that way in another feed just like this). It's a new feature DevOps added for this very purpose, and was first enabled for the VSSDK and we asked to have the same for our stuff.